README.md

```diff
@@ -99,32 +98,50 @@ color you like:
 
 ## Directory tracking
 
+The `default-directory` would change when you change your working directory.
+and text in the vterm buffer has a text property `working-directory`.
+
+
+
+vterm enable `compilation-shell-minor-mode` by default.
+even you change your working directory, `next-error` `previous-error` still
+can find out the corresponding source code.
```

Comment:
This has to be reformulated to something like

```With `compilation-shell-minor-mode` enabled, you can jump to file references via `next-error` and `previous-error` like in a `compilation-mode` buffer.```
The technical details don't belong into the README.